### PR TITLE
Added: dependabot.yml

### PR DESCRIPTION
**Pull request by GoliathLabs**
Good day!
I have added a proper dependabot.yml. Dependabot now takes care of gh-action and docker updates :)

### CODE REVIEW

**Comment by kereis** [comment]
Hey, thank you for your pull request. 👍 
Is there any way how to test it or is it something we'll find out when time passes? I haven't really used dependabot before. :s

**Comment by GoliathLabs** [comment]
It will create pull requests with the changes suggested by dependabot. It will probably create some after this PR is merged and after that it will take care of monthly updates

**Comment by kereis** [comment]
Alright. Also, we should maybe add a `.dockerignore` file so files in `.github` are ignored when building Docker images.

**Comment by GoliathLabs** [comment]
Done. Thank you for pointing that out :)

**Comment by GoliathLabs** [comment]
Shall we merge it?

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.dockerignore
+.git
+.github
+.gitignore
+.gitmodules
+Dockerfile
+docker-compose.yml
```
Comment (kereis) [inline .dockerignore:7]: Does this work recursively? Like `Dockerfile` in `/docker/` and `/alpine/` are also ignored?

Comment (kereis) [reply .dockerignore:7]: According to https://docs.docker.com/engine/reference/builder/#dockerignore-file, we might need to prepend `**/` so ignoring these files work recursively.

Comment (kereis) [reply .dockerignore:7]:
```suggestion
**/.dockerignore
.git
.github
.gitignore
.gitmodules
**/Dockerfile
**/docker-compose.yml
```

**Comment by GoliathLabs** [comment]
Commited the changes :)

**Pull request merged**